portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -1729,14 +1729,26 @@ definition {
 	}
 
 	macro viewRichTextPredefinedValue {
-		SelectFrame.selectFrameNoLoading(
-			key_fieldLabel = "Rich Text",
-			locator1 = "CKEditor#BODY_FIELD_IFRAME_WEB_CONTENT_ARTICLE");
+		if (isSet(fieldLabel)) {
+			SelectFrame.selectFrameNoLoading(
+				key_fieldLabel = "${fieldLabel}",
+				locator1 = "CKEditor#BODY_FIELD_IFRAME_WEB_CONTENT_ARTICLE");
 
-		AssertTextEquals(
-			key_fieldName = "Rich Text",
-			locator1 = "CKEditor#BODY",
-			value1 = "${fieldValue}");
+			AssertTextEquals(
+				key_fieldName = "${fieldLabel}",
+				locator1 = "CKEditor#BODY",
+				value1 = "${fieldValue}");
+		}
+		else {
+			SelectFrame.selectFrameNoLoading(
+				key_fieldLabel = "Rich Text",
+				locator1 = "CKEditor#BODY_FIELD_IFRAME_WEB_CONTENT_ARTICLE");
+
+			AssertTextEquals(
+				key_fieldName = "Rich Text",
+				locator1 = "CKEditor#BODY",
+				value1 = "${fieldValue}");
+		}
```

Comment:
@leticiachiu, instead of repeat the instructions `SelectFrame` and `AssertTextEquals` how about just use a variable to get the proper label?
```
var key_fieldLabel;

if (isSet(fieldLabel)) {
    key_fieldLabel = "${fieldLabel}";
}
else {
    key_fieldLabel = "Rich Text";
}

SelectFrame.selectFrameNoLoading(
    key_fieldLabel = "${key_fieldLabel}",
    locator1 = "CKEditor#BODY_FIELD_IFRAME_WEB_CONTENT_ARTICLE");

AssertTextEquals(
    key_fieldName = "${key_fieldLabel}",
    locator1 = "CKEditor#BODY",
    value1 = "${fieldValue}");
```
What do you think?